### PR TITLE
[SNMP] Ping instance config should override init_config settings

### PR DESCRIPTION
**Pull request by ken-schneider**
### What does this PR do?
Fixes behavior where `init_config` for ping would override any instance level config set for a device. It should be the other way around, e.g. if ping is generally enabled at the `init_config` level, but an instance has ping disabled, that instance should have ping disabled. Before this change, it would still be enabled.

### Motivation
Noticed the behavior during some local testing. While this behavior is technically valid, it's at odds with the way other configs inside the SNMP integration work today.

### CODE REVIEW

File: releasenotes/notes/fix-ping-config-f4339de3a55bff87.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix issue where for ping init config took priority over instance config
```
Comment (hestonhoffman) [inline releasenotes/notes/fix-ping-config-f4339de3a55bff87.yaml:11]:
```suggestion
    Fix issue where init config for ping took priority over instance config.
```
Hopefully this hasn't changed the meaning. It was a little confusing as written

**Review by hestonhoffman — COMMENTED** [review]
Small release note edit for clarity

**Review by zoedt — APPROVED** [review]

File: releasenotes/notes/fix-ping-config-f4339de3a55bff87.yaml (hunk 1 above)
Comment (ken-schneider) [reply releasenotes/notes/fix-ping-config-f4339de3a55bff87.yaml]: It hasn't changed the meaning, it's definitely more clear with the edit.

**Review by domalessi — APPROVED** [review]

**Pull request merged**